lib/llvm-backend/Cargo.toml

```diff
@@ -18,8 +18,8 @@ libc = "0.2.60"
 byteorder = "1"
 
 [dependencies.inkwell]
-git = "https://github.com/wasmerio/inkwell"
-branch = "llvm8-0"
+git = "https://github.com/TheDan64/inkwell"
+rev = "781620e9fa30e51a6e03bd0d49b5f5bb7a782520"
```

Comment:
Can we update our local inkwell repo and use it instead?

Reply:
Of course we can, but I thought the idea was to move to master as that will match what gets uploaded to crates.io?

Reply:
Yeah, that would be even better!